### PR TITLE
Merge master to staging

### PR DESCRIPTION
**Pull request by ailisp**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -4,3 +4,4 @@
 
 /runtime/ @nearmax @evgenykuzyakov
 
+/runtime/ @nearmax @evgenykuzyakov
```
Comment (ilblackdragon) [inline CODEOWNERS]: duplicate

File: runtime/runtime/Cargo.toml [hunk 2]
```diff
@@ -15,16 +15,20 @@ kvdb = "0.1"
 rand = "0.6"
 rand_xorshift = "0.1"
 sodiumoxide = "0.2.5"
+ethash = "0.3"
```
Comment (ilblackdragon) [inline runtime/runtime/Cargo.toml]: this and ethereum-bigint below is not needed anymore, I think

**Comment by ailisp** [comment]
@bowenwang1996 @nearmax @janedegtiareva @vgrichina Any reviews?

File: CODEOWNERS [hunk 3]
```diff
@@ -3,4 +3,3 @@
 /chain/ @ilblackdragon @SkidanovAlex
 
 /runtime/ @nearmax @evgenykuzyakov
-
```
[the remark this answers is not in the document]
Comment (ailisp) [reply CODEOWNERS]: having line 5 duplicate by merging, then line 6 was removed by mistake

File: chain/jsonrpc/src/lib.rs [hunk 4]
```diff
@@ -1,30 +1,36 @@
 #![feature(await_macro, async_await)]
+#[macro_use]
+extern crate lazy_static;
```
[the remark this answers is not in the document]
Comment (ailisp) [reply chain/jsonrpc/src/lib.rs]: yup

File: CODEOWNERS (hunk 3 above)
[the remark this answers is not in the document]
Comment (ailisp) [reply CODEOWNERS]: wtf. i do have line 6 locally :thinking:

[the remark this answers is not in the document]
Comment (ailisp) [reply CODEOWNERS]: image

**Pull request merged**